code/src/components/Form.js

```diff
@@ -0,0 +1,238 @@
+									click={nextQuestion}
+									text={name ? 'Next' : 'Type name'}
+								/>
+							</div>
+						</section>
+					)}
+
+					{question === 1 && (
+						<section className="question-container">
+							<InputSelect
+								question="What's your favourit season?"
+								setSeason={setSeason}
+								value={season}
+							/>
+							<div className="navigation">
+								<Button button="button" click={previousQuestion} text="Back" />
+								<Button
+									button="button"
+									disabled={!season}
+									click={nextQuestion}
+									text={season ? 'Next' : 'Select season'}
+								/>
+							</div>
+						</section>
+					)}
+					{question === 2 && (
+						<section className="question-container">
+							<InputCheckbox
+								question="Select which months you prefer"
+								selectedMonths={months}
+								onMonthsChange={handleMonthsChange}
+								array={
+									seasonArray[
+										seasonArray.findIndex(item => item.season === season)
+									].months
+								}
+							/>
+							<div className="navigation">
+								<Button button="button" click={previousQuestion} text="Back" />
+								<Button
+									button="button"
+									disabled={!months}
+									click={nextQuestion}
+									text="Next"
+								/>
+							</div>
+						</section>
+					)}
+					{question === 3 && (
+						<section className="question-container">
+							<InputRange
+								question={`How much do you like ${season}? (scale 1-10)`}
+								value={value}
+								setValue={setValue}
+								step={1}
+							/>
+							<div className="navigation">
+								<Button button="button" click={previousQuestion} text="Back" />
+								<Button
+									button="button"
+									disabled={value <= 0}
+									click={nextQuestion}
+									text={value ? 'Next' : 'Slide range'}
+								/>
+							</div>
+						</section>
+					)}
+
+					{question === 4 && (
+						<section className="question-container">
+							<InputRadio
+								question={`${season} what a great season, how many have you experienced?`}
+								array={ageGroups}
+								setAge={setAge}
+								age={age}
+							/>
+							<div className="navigation">
+								<Button button="button" click={previousQuestion} text="Back" />
+								<Button
+									button="button"
+									disabled={!age}
+									click={nextQuestion}
+									text={age ? 'Next' : 'Select age'}
+								/>
+							</div>
+						</section>
+					)}
+					{question === 5 && (
+						<section className="question-container-summary">
+							<Summary
+								name={name}
+								season={season}
+								months={months}
+								selected={age}
+								value={value}
+							/>
+							<div className="email-container">
+								<InputEmail
+									question="Type your email for a summary!"
+									value={email}
+									setEmail={setEmail}
+								/>
+								<div className="navigation">
+									<Button
+										button="button"
+										click={previousQuestion}
```

Comment:
Great with a previous button so the user can go back if they want to

Reply:
😀